src/core/qgsjsonutils.h

```diff
@@ -315,7 +315,9 @@ class CORE_EXPORT QgsJsonUtils
      * \see stringToFields()
      * \note this function is a wrapper around QgsOgrUtils::stringToFeatureList()
      */
+#ifdef SIP_PYQT5_ONLY
```

Comment:
This reads awkwardly. Could we use "ifdef SIP_PYQT5_RUN" instead?

Reply:
Done